assets/scss/core/_typography.scss

```diff
@@ -23,6 +23,7 @@ $ds-headers-h3-font-size: 30px;
 $ds-headers-h4-font-size: 22px;
 $ds-headers-h5-font-size: 18px;
 $ds-headers-h6-font-size: 13px;
+$ds-font-size-small: 13px;
```

Comment:
it's already use 13px. isn't better in `$ds-headers-h6-font-size: $ds-font-size-small;` ?

Reply:
Las variables las voy a ir limpiando y seguramente agregando. Solo es una minilimpieza de los estilos que ya estaban.